Task 3/StackQueues

```diff
@@ -0,0 +1,114 @@
+package stacksqueues
```

Comment:
Rename file with .kt extension

Reply:
Folder structure created for Stackqueues Using Kotlin provided.Approve the PR upon review